components/BackToTop/BackToTopButton.tsx

```diff
@@ -47,6 +47,7 @@ export const BackToTopButton = () => {
       <button
         className="focus:animate-button-press rounded-full border border-white bg-violet-600 p-4 text-white shadow-xl duration-300 transition-colors focus:ring group-hover:border-dashed group-hover:border-violet-400 group-hover:bg-white dark:drop-shadow-[5px_5px_8px_rgba(124,58,237,0.25)] dark:group-hover:bg-[#101623] md:border-violet-600"
         onClick={handleClick}
+        title="Back to top"
```

Comment:
When an image contains no text description, neither assistive technology such as screenreaders nor crawlers such as search engine bots will know what the button does. Adding a `title` is one way to indicate the purpose of the image. This holds true for buttons whose only content is an image.

https://www.w3.org/TR/WCAG20-TECHS/H37.html

Reply:
Ohh, thanks